src/Api/OperationMethodResolverInterface.php

```diff
@@ -16,13 +16,15 @@
 use ApiPlatform\Core\Exception\RuntimeException;
 
 /**
- * Resolves the HTTP method associated with an operation.
+ * Resolves the uppercased HTTP method associated with an operation.
  *
  * @author Kévin Dunglas <dunglas@gmail.com>
  */
 interface OperationMethodResolverInterface
 {
     /**
+     * Resolves the uppercased HTTP methods associated with a collection operation.
```

Comment:
methods -> method

Also, I feel we don't need to say "uppercased" explicitly. Because that should be the expectation for HTTP verbs?

Reply:
I do that because in Symfony the PHPDoc specifies `uppercased` for each method that returns an HTTP verb.